module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -575,7 +603,8 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                     'issues' => $holdingsStatements,
                     'supplements' => $holdingsSupplements,
                     'indexes' => $holdingsIndexes,
-                    'callnumber' => $holding->callNumber ?? '',
+                    'callnumber' => $callNumberData['callnumber'],
```

Comment:
If you make `chooseCallNumber` return a `callnumber_prefix` key instead of just `prefix`, then you could eliminate these two lines by using `$items[] = $callNumberData + [ ...` above.